SchedulerCreator/src/js/exporters/HTML/HTMLExporter.js

```diff
@@ -7,6 +7,7 @@ module.exports = (timedScheduleObject) => {
         $('table thead tr').append(`<th class="letter">${day}</th>`);
     });
     const numOfClasses = Object.values(timedScheduleObject)[0].length;
+    const colors = ["tomato", "dodgerblue", "yellow", "orange", "mediumseagreen", "lightblue", "pink", "#d6ff99", "gainsboro", "#FAE7B5", "#a89de1", "burlywood"]
```

Comment:
Should be globally declared (same array each time)

Reply:
Okay